pkg/sql/sqlbase/namespace.go

```diff
@@ -227,3 +228,16 @@ func LookupDatabaseID(
 ) (bool, ID, error) {
 	return LookupObjectID(ctx, txn, codec, keys.RootNamespaceID, keys.RootNamespaceID, name)
 }
+
+// WriteNameKeyToBatch writes a name key to a batch and returns the batch.
+func WriteNameKeyToBatch(
```

Comment:
I was thinking instead a function on the planner that takes in a `sqlbase.DescriptorKey`. It'll handle the tracing enabled check, and then just run the batch as well.

Reply:
Done